static/styles/portico/markdown_dev_panel.css

```diff
@@ -0,0 +1,140 @@
+button {
+    background-color: hsl(152, 52%, 91%);
+    color: hsl(146, 50%, 36%);
+}
```

Comment:
We should write these CSS rules to be contained to only apply to this page.

Even if it's in its own bundle and thus can only impact that page, it's not a good coding pattern.

Could all the CSS in this file by inside `#markdown-panel`?

Reply:
I rearranged CSS rules in their respective parent selector. Now most of the rules are inside `#markdown-panel` but some rules for navbar and spacing needs to be outside, 
```css
.top-navbar {
    display: grid;
    grid-template-columns: 1fr 5fr 1fr;
}

.centerpiece {
    font-size: 30px;
    text-align: center;
}

.pad-small {
    height: 100px;
    width: 100%;
}

.pad-tiny {
    height: 50px;
    width: 100%;
}
```
P.S: There is one fix for this - We can wrap `portico_content` in `#markdown_dev_panel_page` and then wrap all css rules inside `#markdown_dev_panel_page`.